File: ios/RNComscoreModule.m
#import <Foundation/Foundation.h>
#import <React/RCTBridgeModule.h>
#import <React/RCTEventEmitter.h>

@interface RCT_EXTERN_MODULE(RNComscoreModule, RCTEventEmitter)

RCT_EXTERN_METHOD(moduleInit:(RCTPromiseResolveBlock)resolve
        rejecter:(RCTPromiseRejectBlock)reject)

RCT_EXPORT_METHOD(trackView:(NSString *) view)

RCT_EXPORT_METHOD(trackEvent:(NSString *)action category:(NSString *)category)

@end

File: ios/RNComscoreModule.swift
import Foundation
import React

@objc(RNComscoreModule)
class RNComscoreModule: NSObject {
  
  @objc(moduleInit:)
  func moduleInit(_ config:NSDictionary) -> Void {
    
    RCTLog("+++ [ComScore] moduleInit");

    //let myPublisherConfig = SCORPublisherConfiguration(builderBlock: { builder in builder?.publisherId = config.value(forKey: "publisherId") as! String })
    //SCORAnalytics.configuration().addClient(with:myPublisherConfig)
    
  }
  
  @objc(trackView:)
  func trackView(_ view:String) -> Void {
    
    
  }

  @objc(trackEvent:category:)
  func resumeAll(_ action:String, category:String) -> Void {
    
    
  }
  
}
